manifests/check.pp

```diff
@@ -116,16 +116,34 @@
 
   $check_name = regsubst(regsubst($name, ' ', '_', 'G'), '[\(\)]', '', 'G')
 
-  file { "/etc/sensu/conf.d/checks/${check_name}.json":
+  case $::osfamily {
+    'windows': {
+      $etc_dir = 'C:/opt/sensu'
+      $conf_dir = "${etc_dir}/conf.d"
+      $user = undef
+      $group = undef
+      $file_mode = undef
```

Comment:
That's a bit of a tricky one. 
Firstly, no the sensu does not create a user the windows service runs as the Local System user.

Secondly, from a puppet point of view the standard unix modal permissions don't really make sense on windows. We have the [puppet-acl](https://forge.puppetlabs.com/puppetlabs/acl) for those people who really care about fine-grained security permissions but that is out-of-scope of this module and would probably be managed at the profile layer. With these things set to undef the SYSTEM user and Administrators group have full permissions and standard Users have read and execute.